### PR TITLE
Update website

### PR DESCRIPTION
**Pull request by mcwimm**
I tried to bring some structure in "getting started" and update the examples....

### CODE REVIEW

File: web/content/en/docs/getting_started/Installation.md [hunk 1]
```diff
@@ -4,150 +4,100 @@ linkTitle: "Installation of pyMANGA"
...
-During the installation a local library with different **Python modules** is installed on your computer.
-These modules contain source code that is difficult to formulate or frequently used - so it does not need to be written in the program, but can be easily retrieved from the library.
-Because the installation differs between operating systems, please select your operating system below.
+The software **pyMANGA** is written in the programming language [Python](https://www.python.org/) and uses Python libraries. 
+That means, during the installation a local library with different **Python modules** will be installed on your computer.
+These modules contain source code that is frequently used by many programmers.
+
+If you are new to programming, it might be convenient to install a user interface, a so-called IDE, to work with Python. 
+Those IDEs make code easier to read and support the installation and management of modules, for example.
+Our team is happy to use [PyCharm](https://www.jetbrains.com/pycharm/) and [Spyder](https://www.spyder-ide.org/), respectively.
```
Comment (jbathmann) [inline web/content/en/docs/getting_started/Installation.md:23]: emacs and vim. :P

File: web/content/en/docs/getting_started/Installation.md [hunk 2]
```diff
@@ -156,96 +106,146 @@ If this is the case, you should get the following output:
...
-<summary>Installation of pyMANGA in Windows</summary>
-<p>
+<summary >Installation of Python Modules in Windows <a name="Installation_Ubuntu"></a></summary>
 
-To be able to run **pyMANGA**, some modules for the **Python Compiler** must be installed.
-You have to open the **prompt** for this.
-You can easily find it by searching for it by typing **"Command Prompt"** and opening it with a **mouse click**.
-Since pyMANGA is a line program, everything happens at the command prompt (see <a href="/docs/getting_started/installation/#Figure_5">Figure 5</a>).
+To install python modules, we use the **"Command Prompt"**.
+You can easily find it by sby typing **"Command Prompt"** in the windows search window and opening it with a **mouse click**.
```
Comment (jbathmann) [inline web/content/en/docs/getting_started/Installation.md:161]:
```suggestion
You can easily find it by typing **"Command Prompt"** in the windows search window and opening it with a **mouse click**.
```

File: web/content/en/docs/getting_started/Installation.md (hunk 1 above)
Comment (pymanga) [reply web/content/en/docs/getting_started/Installation.md:23]: 👎

**Review by jbathmann — APPROVED** [review]
:+1:

**Pull request merged**